### DISCUSSION TITLE
Ignore single character differences

### DISCUSSION THREAD

**Discussion by Th3C0D3R** [WinMerge/winmerge#1769] [General]
Is there a way to only show lines where differences are longer than X (in my case 1 or two) character?
As I got a hugh js-file (18MB) with over 500k lines and 80% are difference, but only with one (or two) changed characters
Example which I want to ignore/hide:
file 1: Hello Boy and Girls
file 2: Hello Boys and Girls

thank you very much for any help or tipps

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have such a feature.

Adding the following to the Substitution filters will cause Boys to be replaced with Boy for comparison, thus ignoring the difference between Boys and Boy.
Note: reversing Boys and Boy will not work as expected. Also, Match whole word only is not working due to a bug.

![image](https://user-images.githubusercontent.com/98126/226148406-f077d1f2-82c9-4799-a014-f51ad64925d9.png)

**Reply by Th3C0D3R** [reply]
Oh ok, than I have to make if work in some other way 
Thanks anyway
